templates/zerver/emails/notify_new_login.source.html

```diff
@@ -33,7 +33,7 @@
 <p>{{ _("If this was you, great! There's nothing else you need to do.") }}</p>
 
 <p>
-    {% trans %}If you do not recognize this login, or think your account may have been compromised, please <a href="{{ realm_uri }}/accounts/password/reset/">reset your password</a> or contact us immediately at <a href="mailto:{{ support_email }}">{{ support_email }}</a>.{% endtrans %}
+    {% trans support_email=macros.email_tag(support_email) %}If you do not recognize this login, or think your account may have been compromised, please <a href="{{ realm_uri }}/accounts/password/reset/">reset your password</a> or contact us immediately at {{ support_email }}.{% endtrans %}
```

Comment:
The `accounts/password/reset` link should be extracted too.

Reply:
Do you mean to just extract out the URL as a variable or create a macro for the `a` tag. 

macro example:
---

```html
{% trans a_tag_start=a_tag_start(realm_uri + "/accounts/password/reset/") support_email=macros.email_tag(support_email) %}
If you do not recognize this login, or think your account may have been compromised, please {{ a_tag_start }}reset your password{{ a_tag_end }}

---
{% macro a_tag_start(url) -%}
    <a href="{{ url }}">
{%- endmacro %}
--
a_tag_end can be defined in context processors as "</a>"


```

Reply:
I have extracted this out as a variable for now.

https://github.com/zulip/zulip/pull/16614/commits/bd4fef43b1bdc93b2eaf1c7bcfdb44be9f423c89